cmd/proxy/Dockerfile

```diff
@@ -1,11 +1,23 @@
-FROM golang:1.11
+FROM golang:1.11-alpine AS builder
 
 RUN mkdir -p $GOPATH/src/github.com/gomods/athens
 WORKDIR $GOPATH/src/github.com/gomods/athens
```

Comment:
`WORKDIR` will create the directory if it doesn't exist, so the preceding `RUN` line is excess.